scss/core/css/utility-classes.css

```diff
@@ -1,3 +1,9 @@
+/**
+ * IMPORTANT: This file is the result of assembling design tokens, contained in the project's "tokens/src" folder
```

Comment:
Should probably remove `contained in the project's "tokens/src" folder` part as these files will be generated in other theme repos also (e.g. brand-edx.org) that will have different file structure and will also override some of the tokens.

Reply:
Corrected, thanks 👍